services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js

```diff
@@ -34,6 +38,10 @@ const COMMAND_NAME = 'transfer';
 
 // eslint-disable-next-line no-unused-vars
 const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
+	indexAccountByAddress(tx.params.recipientAddress);
```

Comment:
L53-L56 seem redundant and can be removed. Also, the revert hook should also be updating both the accounts/transactionsTable.